### PR TITLE
feat: Card implemented

### PR DESCRIPTION
**Pull request by govardhanreddyc79**

### CODE REVIEW

File: Components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      margin-top: 1rem;
+  }
+    
+  .card {
+      margin:1rem 1rem 2rem 1rem;
+      background-color: white;
+      display: flex;
+      width: 28rem;
+      flex-direction: column;
+      box-shadow: 1px 2px 5px 2px;
```
Comment (parvez-rashid) [inline Components/Card/card.css]: There is no color here, color is a required parameter for box-shadow.

File: Components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      display: flex;
+      position: relative;
+      min-height: 15rem;
+      flex-direction: row;
+    }
+    
+    .card-img-hz {
+      width:12.5rem;
+      height: 10rem;
+      border-radius:1rem 0rem 0rem 0rem
```
Comment (parvez-rashid) [inline Components/Card/card.css]: Semi-colon is missing here

**Review by parvez-rashid — APPROVED** [review]
I've left my comments below, please have a look. Also, include a preview of the component next time.

**Comment by Wsn-195** [comment]
Looks good and you have consistent in code style .

**Review by aurobindoGupta — APPROVED** [review]

File: Components/Card/card.css (hunk 1 above)
Comment (govardhanreddyc79) [reply Components/Card/card.css]: sure, will look into that

File: Components/Card/card.css (hunk 2 above)
Comment (govardhanreddyc79) [reply Components/Card/card.css]: will check this, Thanks

**Pull request merged**